packages/react-select/src/Select.js

```diff
@@ -363,29 +487,49 @@ export default class Select extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
     const { value } = props;
-    this.cacheComponents = memoizeOne(this.cacheComponents, isEqual).bind(this);
```

Comment:
I removed the memoization here because the memoization seems unnecessary (`cacheComponents` just ends up doing an object spread). It is now a `getComponents` method. The memoization was of the components was originally introduced [here](https://github.com/JedWatson/react-select/pull/2814).

Reply:
I actually ended up adding back the memoization in a latter commit, but still moved it be a getter instead of a class instance field.